packages/js/storybook/package.json

```
@@ -0,0 +1,31 @@
+{
+	"name": "woocommerce-grow-story-book",
+	"title": "WooCommerce Grow - Storybook Package",
+  	"type": "module",
+	"license": "GPL-3.0-or-later",
+	"version": "1.12.5",
```

I think we should start versioning from scratch. I guess it's just leftover from GLA, isn't it?
```suggestion
	"version": "0.0.1",
```